az-release-pipelines/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN apk update \
 
 LABEL version="1.0.0"
 
-LABEL maintainer="Nishu Bansal" 
+LABEL maintainer="Microsoft Corporation" 
 LABEL com.github.actions.name="GitHub Action to trigger Azure release pipeline" 
```

Comment:
@nishubansal Perhaps we could remove the word `Github Action` from the name to make it consistent with other actions?